### DISCUSSION TITLE
Not clear search value in autcomplete on blur/focus change

### DISCUSSION THREAD

**Discussion by simenkf** [vuetifyjs/vuetify#18588] [Q&A]
I have a `v-autocomplete` where I lookup the typed search value (an address) as the user types (calling an AP, fetching addresses used for the items array). Whenever a user switches window or clicks outside the field (aka a blur/focus event), the search value is deleted/cleared. A use-case for us is that the we start typing the address, then switch to some other window or browser tab to check/recall what the full address was, and then return back to the app to complete the search query. However, now you need to start over (although previously fetched addresses are still in the items array).

How can i make it so that the search query is only deleted when an item from the item list is clicked/selected?

I know `v-combobox` perhaps could solve this issue. However, I need the model-value to (an address object) to only ever be an item from the items-array, and with `v-combobox` it changes the value is I type.

**Comment by KaelWD** [comment]
Use a combobox and pass the selection array through `.filter(item => typeof item !== string)`?
Or save the previous search value in another variable on blur and set it back on focus.

**Comment by martijnimhoff** [comment]
The proposed solution did not work for me. I used `v-model:search` and `@blur` to get the search value on blur. This works, however on focus i put the value back via the same `v-model:search`. This has no effect. 

@simenkf how did you solve this in the end?

**Comment by RishiDiwanTT** [comment]
Setting Autocomplete search value on focus is not working, since it seems the `@update:focused` callback runs before the internal callback that clears out the value.
